es-design-system/pages/molecules/es-support.vue

```diff
@@ -39,7 +39,7 @@
     </div>
 </template>
 <script>
-import benr from '@/assets/imgs/benr.png';
+import benr from '@/assets/img/benr.png';
```

Comment:
Can we swap this with something like https://place-hold.it/

Reply:
Can't quite use a full square image like that because the different background colors/gradients need to be able to show through for people to be able to see the difference between the variants. Changed to use the IconPerson SVG, but as an imported image asset, so we can pass it into image `src` props.